client/src/components/questions/Answer.jsx

```diff
@@ -74,11 +84,18 @@ const Answer = ({ answerData }) => {
     }
   }
 
+  var imageClick = (e) => {
+    setModalURL(e.target.src);
+    setImageModal(true);
+  }
+
   return (
     <StyledAnswer >
       <div>
         {answerData.body}
       </div>
+      {answerData.photos.length > 0 ? answerData.photos.map((url) => (<StyledAnswerImage src={url} key={url} onClick={imageClick}></StyledAnswerImage>)) : <div></div>}
+      {imageModal ? <ImageModal setImageModal={setImageModal} url={modalURL}/> : <div></div>}
```

Comment:
I am certain you would have noticed this if it didn't work. But don't you have to return what is in the parenthesis in the map function to make it work?

Reply:
not if it is in one line, heres a good resource to look at: https://mariusschulz.com/blog/returning-object-literals-from-arrow-functions-in-javascript